### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by RiaanLab**
My first stab at the new README.md. The license version still needs to be updated though.

### CODE REVIEW

**Comment by Arcanemagus** [comment]
The license is specified in the `package.json` as MIT. Also if you are going to specify it any further than that it should be in a `LICENSE.md`, not in the readme.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
 # linter-swagger
 
-This plugin for [Linter](https://github.com/AtomLinter/Linter) will lint [Swagger specifications](http://swagger.io/), both JSON and YAML.
+## Synopsis
+
+This plugin is a plugin for an [Atom](https://atom.io/) text editor plugin called [Linter](https://github.com/AtomLinter/Linter). Linter is used to lint various text formats in Atom and linter-swagger will specifically lint [Swagger specification](http://swagger.io/specification/) files. linter-swagger will validate Swagger files in both JOSN and YAML formats.
```
Comment (Arcanemagus) [inline README.md]: "plugin" -> "package" (except the second instance).

**Comment by Arcanemagus** [comment]
Can you put the maximum line length to under 80 chars? You might want to install https://github.com/AtomLinter/linter-markdown :wink:

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+
+## Motivation
+
+Swagger is gaining ground as a standard to write YAML or JSON based API specifications. To validate the YAML or JSON files one could use [various tools](http://swagger.io/swagger-editor/) to edit and validate the text file against the Swagger specification. Atom is a great tool to do YAML or JSON editing in and by using linter-swagger you can validate and write the Swagger specifications all form one application.
```
Comment (Arcanemagus) [inline README.md]: "form" -> "from"

File: README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+
+## Motivation
+
+Swagger is gaining ground as a standard to write YAML or JSON based API specifications. To validate the YAML or JSON files one could use [various tools](http://swagger.io/swagger-editor/) to edit and validate the text file against the Swagger specification. Atom is a great tool to do YAML or JSON editing in and by using linter-swagger you can validate and write the Swagger specifications all form one application. 
+
+## Installation
+
+This plugin, linter-swagger, can be enabled via the Atom GUI or command line. Please see this [link to enabling packages in Atom] (https://atom.io/docs/latest/using-atom-atom-packages). 
+
+To enable the linter-swagger plugin you will have install the Linter plugin (which is a prequisite to using linter-swagger in Atom) and addiotionally you will need to install the linter-sawgger plugin.
```
Comment (Arcanemagus) [inline README.md]: addiotionally -> additionally

Comment (Arcanemagus) [inline README.md]: Also, this part will become unnecessary once https://github.com/AtomLinter/linter-swagger/issues/16 is implemented :wink:.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+
+linter-swagger will know which files to validate based on the swagger version line (normally) at the top in the Swagger file. linter-swagger will lint either JSON or YAML files. 
+
+When using the linter-swagger plugin '''Please note:''' that you need to place the Swagger version number (for swagger version 2 this is a '''2''' as in swagger: "2") in double qoutes so that linter-swagger will know which JSON or YAML files to start validating. Ex.
+
+swagger: "2"
+
+and '''NOT''' 
+
+swagger: '2'
```
Comment (Arcanemagus) [inline README.md]: It might be a good idea to make the second set of quotes optional, as they don't look right in a YAML file.

**Comment by RiaanLab** [comment]
Damn man!!! Late night. Thanks for fixing all the spelling mistakes.

File: README.md (hunk 4 above)
Comment (RiaanLab) [inline README.md]: Agree, but as you mentioned the code requires it.

**Comment by spoike** [comment]
How come this was closed?

**Comment by RiaanLab** [comment]
I was not supposed to create another branch.

Riaan Labuschagne

riaan@fullfacing.com
Cel: +2783 4444148
Tel: +2721 5213000
Fax: +2721 521 3001
www.fullfacing.com

**Comment by spoike** [comment]
Ok, cool. :wine_glass: 

I'll add the option to write `swagger: '2.0'` in yaml files as I was not aware you could do that. All examples I've seen so far were using double quotes.

**Comment by spoike** [comment]
BTW, I've updated the README file with the bare essentials.

**Pull request closed without merging**